go.mod

```diff
@@ -4,25 +4,28 @@ go 1.13
 
 require (
 	github.com/RoaringBitmap/roaring v0.4.23
-	github.com/blevesearch/bleve_index_api v0.0.0-20201109204916-3d41c3915248
+	github.com/blevesearch/bleve_index_api v0.0.1
 	github.com/blevesearch/blevex v0.0.0-20190916190636-152f0fe5c040
+	github.com/blevesearch/cld2 v0.0.0-20200327141045-8b5f551d37f5 // indirect
```

Comment:
Why this new indirect dependency (also the other two: kagome.ipadic and snowball) ? From this change it's not quite obvious to me what uses this.

Reply:
nothing with go modules is obvious with  me.  `go mod tidy` adds it.  to me the real question is why hasn't it been there all along?

Reply:
Ah, I just checked that we do import this with blevex which is imported within bleve ..

```
./blevex/detectlang/detect_lang_filter.go:	"github.com/blevesearch/cld2"
```

Now I do see this entry already in the go.mod of blevesearch/bleve on master, I'mma guess that you removed it accidentally in one of the earlier changes :)

Reply:
blevex is, but not the indirects.   they have not been removed in earlier changes.

Reply:
Shoot, you're right, I was looking at my local. I must've run `go mod tidy` at some point, this is what I have ..

```
[18:29:29] AD: ~/Documents/go/src/github.com/blevesearch/bleve $ git diff
diff --git a/go.mod b/go.mod
index 10f4c571..f9d83f69 100644
--- a/go.mod
+++ b/go.mod
@@ -5,6 +5,7 @@ go 1.13
 require (
        github.com/RoaringBitmap/roaring v0.4.23
        github.com/blevesearch/blevex v0.0.0-20190916190636-152f0fe5c040
+       github.com/blevesearch/cld2 v0.0.0-20200327141045-8b5f551d37f5 // indirect
        github.com/blevesearch/go-porterstemmer v1.0.3
        github.com/blevesearch/segment v0.9.0
        github.com/blevesearch/snowballstem v0.9.0
@@ -15,12 +16,23 @@ require (
        github.com/blevesearch/zap/v15 v15.0.2
        github.com/couchbase/moss v0.1.0
        github.com/couchbase/vellum v1.0.2
+       github.com/cznic/b v0.0.0-20181122101859-a26611c4d92d // indirect
+       github.com/cznic/mathutil v0.0.0-20181122101859-297441e03548 // indirect
+       github.com/cznic/strutil v0.0.0-20181122101858-275e90344537 // indirect
+       github.com/facebookgo/ensure v0.0.0-20200202191622-63f1cf65ac4c // indirect
+       github.com/facebookgo/stack v0.0.0-20160209184415-751773369052 // indirect
+       github.com/facebookgo/subset v0.0.0-20200203212716-c811ad88dec4 // indirect
        github.com/golang/protobuf v1.3.2
+       github.com/ikawaha/kagome.ipadic v1.1.2 // indirect
+       github.com/jmhodges/levigo v1.0.0 // indirect
        github.com/kljensen/snowball v0.6.0
        github.com/rcrowley/go-metrics v0.0.0-20190826022208-cac0b30c2563
+       github.com/remyoudompheng/bigfft v0.0.0-20200410134404-eec4a21b6bb0 // indirect
        github.com/spf13/cobra v0.0.5
        github.com/steveyen/gtreap v0.1.0
        github.com/syndtr/goleveldb v1.0.0
+       github.com/tebeka/snowball v0.4.2 // indirect
+       github.com/tecbot/gorocksdb v0.0.0-20191217155057-f0fad39f321c // indirect
        github.com/willf/bitset v1.1.10
        go.etcd.io/bbolt v1.3.5
        golang.org/x/text v0.3.0
```

Reply:
From an online source ..

An // indirect comment indicates that no package from the required module is directly imported by any package in the main module. The go command adds an indirect requirement when the selected version of a module is higher than what is already implied (transitively) by the main module's other dependencies.

Reply:
And do you know why the selected version is different (or higher) than what is implied transitively?

Reply:
No, I haven't checked what's changed.

Reply:
Again, I don't think anything has changed.  I think we just hadn't been running `go mod tidy` on the committed files.

Reply:
I see, ok then.

Reply:
The answer to my question does not pertain to a change, but to an understanding of Go modules and our code.  Like I understand what all those words mean, but I don't understand what would cause that.  Like if you had a sample project, what would you do to make `go mod tidy` add the lines with "// indiriect"

Reply:
From the little that I know - I think this is because of the absentee go.mod within blevesearch/blevex.

Reply:
Merging this: https://github.com/blevesearch/blevex/pull/48 and updating the blevex version here, will address this.